README.md

```diff
@@ -181,7 +182,23 @@ Before starting this codelab, make sure that you've installed:
         * listens for added ICE candidates from the remote peer and adds them to its `RTCPeerConnection` instance. 
     It is important when listening to database changes to filter out anything that isn't a new addition, since we otherwise would have added the same set of ICE candidates over and over again.
     
-    Complete this step by uncommenting the calls to this function in both the `joinRoomById` and `createRoom` methods.
+    Complete this step by uncommenting the calls to this function in both the `joinRoomById` and `createRoom` methods; you can find them after `// Uncomment to collect ICE candidates below`.
+
+1. __Try it out__
+
+    1. Open a new browswer tab at http://localhost:5000, or refresh the tab you opened above.
+    1. Mute your speakers to avoid loud piercing feedback!
```

Comment:
bahaha! so true.  I had this on in the living room and forgot that would happen.  It was....shocking. 😝